lib/pbench/cli/server/shell.py

```diff
@@ -1,8 +1,13 @@
 #!/bin/env python3
 
 from pbench.server.api import create_app
+import subprocess
 
 
 def main():
     app = create_app()
-    app.run(debug=True, port=app.config["PORT"])
+    port = app.config["PORT"]
+    host = app.config["GUNICORN_HOST"]
+    workers = app.config["GUNICORN_WORKERS"]
+    subprocess.run(["gunicorn", "--workers", str(workers), "--bind",
```

Comment:
Looks good.  Just wondering if the gunicorn writes its own pid to a file that we can that use to kill the specific gunicorn pid properly, see https://stackoverflow.com/questions/14604653/how-to-stop-gunicorn-properly.

Reply:
Thanks for the link, good idea we can write the gunicorn pid to a file and close it when necessary